cellfinder_napari/detect.py

```diff
@@ -19,107 +27,50 @@
 
 
 def detect():
-    from math import ceil
-
-    # from fancylog import fancylog
-    # import cellfinder_napari as program_for_log
-    from napari.qt.threading import thread_worker
-    from cellfinder_core.main import main as cellfinder_run
-    from cellfinder_core.classify.cube_generator import get_cube_depth_min_max
-    from imlib.cells.cells import Cell
-    from .utils import cells_to_array
-
-    DEFAULT_PARAMETERS = dict(
-        voxel_size_z=5,
-        voxel_size_y=2,
-        voxel_size_x=2,
-        Soma_diameter=16.0,
-        ball_xy_size=6,
-        ball_z_size=15,
-        Ball_overlap=0.6,
-        Filter_width=0.2,
-        Threshold=10,
-        Cell_spread=1.4,
-        Max_cluster=100000,
-        Trained_model=Path.home(),
-        Start_plane=0,
-        End_plane=0,
-        Number_of_free_cpus=2,
-        Analyse_local=False,
-        Debug=False,
-    )
-
     @magicgui(
-        header=dict(
-            widget_type="Label",
-            label=f'<h1><img src="{brainglobe_logo}"width="100">cellfinder</h1>',
-        ),
-        detection_label=dict(
-            widget_type="Label",
-            label="<h3>Cell detection</h3>",
-        ),
-        data_options=dict(
-            widget_type="Label",
-            label="<b>Data:</b>",
-        ),
-        detection_options=dict(
-            widget_type="Label",
-            label="<b>Detection:</b>",
-        ),
-        classification_options=dict(
-            widget_type="Label",
-            label="<b>Classification:</b>",
-        ),
-        misc_options=dict(
-            widget_type="Label",
-            label="<b>Misc:</b>",
+        header=html_label_widget(
+            f'<img src="{brainglobe_logo}"width="100">cellfinder', "h1"
```

Comment:
is this correct? Looks odd that the `img` tag isn't closed, but it might just be how this works

Reply:
Doubt it, probably me just chucking rubbish html into the repo.